projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1063,21 +1063,10 @@ public function should_submit_post_pre_checks( $post ) {
 			$submit_post = false;
 		}
 
-		// On quick edit, autosave, etc but do fire on p2, quickpress, and instapost ajax.
 		if (
-			defined( 'DOING_AJAX' )
+			defined( 'DOING_AUTOSAVE' )
 		&&
-			DOING_AJAX
-		&&
-			! did_action( 'p2_ajax' )
-		&&
-			! did_action( 'wp_ajax_json_quickpress_post' )
-		&&
-			! did_action( 'wp_ajax_instapost_publish' )
-		&&
-			! did_action( 'wp_ajax_post_reblog' )
-		&&
-			! did_action( 'wp_ajax_press-this-save-post' )
+			DOING_AUTOSAVE
```

Comment:
This is tricky as we can't be sure what AJAX requests will now pass this check, which didn't before. I think we'll need a `|| did_action( 'wp_ajax_inline_save' )` check added, because otherwise this would start firing on Quick Edits.

The bulk edits are still covered by the later check, and that's what #23976 was fixing.

Publicize should only be triggered as posts move to the `publish` or `future` status anyway, otherwise the `is_publicizeable` check will fail. So other AJAX requests that edit the title of the post or something, won't be an issue.